packages/webapp/utils/numberAbreviation.ts

```diff
@@ -0,0 +1,10 @@
+export const abbreviatedBalance = (balance: any) => {
+    const THOUSAND = 1000;
```

Comment:
prob not needed as a variable

Reply:
please be more specific or suggest the change, I am learning that stuff , you mean that I dont need to use const here?

Reply:
sorry. yea i'm just suggesting that having a variable named 1000 seems kind of redundant. can just use the literal value 1000

Reply:
totally , I just updated that, I pushed to my repo, trying to figure out if I need to update this commit of if its automatic